.yamato/webgl-build.yml

```diff
@@ -0,0 +1,39 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+{% for project in projects -%}
+{% if project.name == "testproject" -%}
+{% for editor in project.test_editors -%}
+build_{{ project.name }}_tests_{{ editor }}_webgl:
+  name: Build {{ project.name }} Tests - {{ editor }} - WebGL
+  agent:
+      type: Unity::VM
+      image: dots-ci/windows10:v1.493-auto
```

Comment:
why not latest stable but specific version?

Reply:
Because we are using dots-ci images, they experiment on their images quite a bit and chances of breaking us with their image updates are more that package-ci images. So we want to tie it down on a version that works. We are doing the same in transport CI.

Reply:
so you're saying `:stable` is actually not so stable? :D